lti_consumer/models.py

```diff
@@ -147,6 +177,14 @@ def clean(self):
             raise ValidationError({
                 "config_store": _("LTI Configuration stores on XBlock needs a block location set."),
             })
+        if self.config_store == self.CONFIG_ON_DB:
+            if self.lti_1p3_tool_public_key == "" and self.lti_1p3_tool_keyset_url == "":
```

Comment:
presumably None would also be a problem here?

also won't this validation error will pop up for everything that was migrated?

Reply:
For all instances of `LtiConfiguration` that have `config_store="CONFIG_ON_DB"`, yes.

Because both model fields have `blank=True` and `null=False` (by default), not setting a value will be stored as the empty string in the database, not `null`. For that reason, `None` is not a concern here.

Reply:
@MichaelRoytman Make sure this doesn't trip with LTI 1.1 configurations.
`lti_1p3_tool_public_key` and `lti_1p3_tool_keyset_url` are only valid for LTI 1.3 integrations, not 1.1 ones.